docs/running-on-yarn.md

```diff
@@ -48,9 +48,9 @@ Most of the configs are the same for Spark on YARN as for other deployment modes
 </tr>
 <tr>
   <td><code>spark.yarn.am.waitTime</code></td>
-  <td>100000</td>
+  <td>100s</td>
```

Comment:
Andrew - I'd prefer to leave out that line. I think it makes things simpler and encourages transitioning to the new paradigm.